pkgs/tools/misc/uhubctl/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, fetchFromGitHub
+, libusb
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uhubctl";
+  version = "unstable-2019-07-31";
+
+  src = fetchFromGitHub {
+    owner = "mvp";
+    repo = "uhubctl";
+    rev = "1961aa02e9924a54a6219d16c61a0beb0d626e46";
+    sha256 = "15mvqp1xh079nqp0mynh3l1wmw4maa320pn4jr8bz7nh3knmk0n1";
+  };
+
+  buildInputs = [ libusb ];
+
+  installFlags = [ "DESTDIR=$(out)" "prefix=" ];
+
+  meta = with stdenv.lib; {
```

Comment:
`platforms` is missing.

Reply:
Fixed in e06d89ab598a3343feb51221dd4367c0ea9683bd